src/test/java/org/generationcp/middleware/dao/dms/DmsProjectDaoIntegrationTest.java

```diff
@@ -520,16 +520,23 @@ public void testGetStudyInstancesWithStudyFilter() {
 		final DmsProject summary =
 			this.createDataset(studyName + " - Summary Dataset", workbenchProject.getUniqueID(), DatasetTypeEnum.SUMMARY_DATA.getId(),
 				study, study);
+		final String externalReferenceId = RandomStringUtils.randomAlphabetic(10);
+		final String externalReferenceSource = RandomStringUtils.randomAlphabetic(10);
```

Comment:
Perform assertion on returned object whether external reference details was returned correctly

Reply:
externalReferenceSource/Id are actually not populated in this method. Those details are populated at the service level in org.generationcp.middleware.service.impl.study.StudyInstanceServiceImpl#getStudyInstancesWithMetadata